playbooks/provisioning/openstack/post-install.yml

```diff
@@ -0,0 +1,57 @@
+---
+- hosts: OSEv3
+  gather_facts: False
+  become: True
+  tasks:
+    - name: Save iptables rules to a backup file
+      when: openshift_use_flannel|default(False)|bool
+      shell: iptables-save > /etc/sysconfig/iptables.orig-$(date +%Y%m%d%H%M%S)
+
+# Enable iptables service on app nodes to persist custom rules (flannel SDN)
+# FIXME(bogdando) w/a https://bugzilla.redhat.com/show_bug.cgi?id=1490820
```

Comment:
@e-minguez so with this w/a, it kinda works with the provider, and doesn't spoil the openshift-ansible roles... The persistence issues gone as well, once I used the os_firewall module as a dependency. So the app nodes end up with the iptables service enabled. The other nodes, like masters and infras, will keep running with iptables disabled and masked (just as it is now, I do not change that).

I really have no better way to deal with all of that mess.